docs/data-sources/node.md

```diff
@@ -29,17 +29,7 @@ data "wallarm_node" "aws_staging" {
 ```
 
 ```hcl
-# Looks up for details on WAF nodes with the specific type (can be "cloud_node" or "node")
-
-data "wallarm_node" "cloud_nodes" {
-  filter {
-    type = "cloud_node"
-  }
-}
-```
-
-```hcl
-# Looks up for the WAF node details by its UUID which is unique over an account
+# Looks up for the Wallarm node details by its UUID which is unique over an account
 
 data "wallarm_node" "example" {
   filter {
```

Comment:
please replace UUID below with the test value, the current one looks like the real one